demos/tests/cases.py

```diff
@@ -591,10 +591,11 @@ def single_option_cases(key, *args):
             model_keys=['-m', '-m_lpr', '-m_va'],
             device_keys=['-d', '-d_lpr', '-d_va'],
             test_cases=combine_cases(
-        TestCase(options={'-no_show': None,
+        TestCase(options={'-no_show': None, '-r' : None, '-n_iqs': '1',
```

Comment:
```suggestion
        TestCase(options={'-no_show': None, '-n_iqs': '1',
```

Reply:
We need this `'-r' : None` to enable the demo outputting the needed raw data.

Reply:
Yeah, but the demo is running with `-r` option for all scenarions by this change. The idea is to run demo with `-r` only if Multi Channel Correctness checking scenario is chosen